lib_test/test_angstrom.ml

```diff
@@ -3,6 +3,11 @@ open Angstrom
 module Alcotest = struct
   include Alcotest
 
+  let bigstring =
+    Alcotest.testable
+      (fun fmt _bs -> Fmt.pf fmt "<bigstring>")
+      ( = )
```

Comment:
This is essentially a `memcmp` on the backing buffer?

Reply:
I think that's the case, though I haven't looked at the implementation.  The docs say that bigarrays support the polymorphic operators - https://caml.inria.fr/pub/docs/manual-ocaml/libref/Bigarray.html

Reply:
Yup confirmed it in the runtime source code as well.